src/snowflake/connector/connection.py

```diff
@@ -1287,6 +1287,7 @@ def cmd_query(
         _update_current_object: bool = True,
```

Comment:
Looking at https://peps.python.org/pep-0249/#connection-objects, this seems like a change which would not affect the pep249 protocol. Should be good to go.